packages/block-editor/src/components/block-pattern-setup/style.scss

```diff
@@ -85,7 +85,7 @@
 		align-items: center;
 		justify-content: space-between;
 		border-top: 1px solid $gray-300;
-		align-self: flex-end;
+		align-self: stretch;
```

Comment:
This change seems to make the toolbar more stable, preventing it from jumping around as slide height varies.